tests/call-centre/protractor.conf.js

```diff
@@ -8,7 +8,7 @@ exports.config = {
       binary: puppeteer.executablePath()
     }
   },
-  baseUrl: "http://django/",
+  baseUrl: "http://cla_frontend/",
```

Comment:
_not a blocker_

💭 I imagine at some point we will have to run these outside the docker-compose network context, where we will have to configure the URLs via `localhost:8001`, etc.

Reply:
:thumbsup: